src/main/kotlin/com/atlassian/performance/tools/jiraactions/w3c/VerboseJsonFormat.kt

```diff
@@ -14,6 +14,7 @@ internal class VerboseJsonFormat {
         Json.createObjectBuilder()
             .add("navigations", navigations.map { serializeNavigationTiming(it) }.toJsonArray())
             .add("resources", resources.map { serializeResourceTiming(it) }.toJsonArray())
+            .add("elements", (elements?.map { serializeElementTiming(it) } ?: emptyList()).toJsonArray())
```

Comment:
`elements` are no longer nullable, so this can become consistent too:
```suggestion
            .add("elements", elements.map { serializeElementTiming(it) }.toJsonArray())
```
IntelliJ is tracking the nullability in Kotlin correctly and even warns about an unnecessary safe call here

Reply:
- [ ] Do it